CAIPs/caip-25.md

```diff
@@ -130,54 +110,17 @@ Example:
 }
 ```
 
-The JSON-RPC method is labeled as `wallet_createSession` and its `params` object contains "requiredScopes" and/or "optionalScopes" objects populated with [CAIP-217][] "scope objects" keyed to [CAIP-217][] scope strings.
-
-- The `requiredScopes` object MUST contain 1 or more `scopeObjects`, if present.
-- The `optionalScopes` object MUST contain 1 or more `scopeObjects`, if present.
+The `sessionScopes` object MUST contain one or more scopeObjects.
 
-A third object is the `scopedProperties` object, which also MUST contain 1 or more objects if present.
-Each object should be keyed to the scope of a `sessionScopes` member to which it corresponds.
-All properties of each object in `scopedProperties` MUST be interpreted by the respondent as proposals or declarations rather than as requirements.
-In addition to making additional properties of or metadata about the corresponding `sessionScopes` member explicit, they can also annotate, support, or extend the negotiation of scope proposals (e.g., providing connection information about unfamiliar scopes, or which accounts to expose to each).
+The `sessionCapabilities` object MAY be present and provides metadata or annotations tied to specific `sessionScopes` keys.
```

Comment:
Seems like we're removing some possibly important language around the optionality of these capabilities? And perhaps should be more specific about what kinds of annotations/metadata we would expect to be put here?